### DISCUSSION TITLE
Where are we at detecting the speakers kind of feature? (Does it exist in ou OUTside of whisper?)

### DISCUSSION THREAD

**Discussion by AIhasArrived** [openai/whisper#1908] [Q&A]
I would appreciate to be able to separate and audio of multiple speakers into multiple audios at least, then I can figure out the transcript for each.
Where are we at now? I saw a discussion from last year but was not sure from where to start and not sure it can work that well for now,
could anyoen englighted me a bit please? Thanks

**Comment by phineas-pta** [comment]
it's called speaker diarization, not a feature of whisper so u must search other models to use
